spring-cloud-task-docs/src/main/asciidoc/features.adoc

```diff
@@ -394,4 +394,16 @@ This can be done either by adding the following annotation to your Task applicat
 ```
 You may also disable Task auto configuration by setting the `spring.cloud.task.autoconfiguration.enabled` property to `false`.
 
-
+[[closing-the-context]]
+=== Closing the Context
+If the application requires the `ApplicationContext` to be closed at the
+completion of a task (all `*Runner#run` methods have been called and the task
+repository has been updated), set the property `spring.cloud.task.closecontext_enabled`
```

Comment:
I am sure there must have been a discussion around naming this property. But, at first look, this property name especially the underscore in `closecontext_enabled` is confusing. Could you help me understand the reasoning of this naming?

Reply:
I can change it to closeContextEnabled.   In this case I was following the previous usage.   
Would that be helpful?